src/main/java/lotto/domain/RewardType.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum RewardType {
+    FIRST(6, 2000000000),
+    SECOND(5,1500000),
+    THIRD(4,50000),
+    FOURTH(3,5000),
+    MISS(-1,0);
+
+    private final int countOfMath;
+    private final int rewardPrice;
+
+    RewardType(int countOfMath, int rewardPrice) {
+        this.countOfMath = countOfMath;
+        this.rewardPrice = rewardPrice;
+    }
+
+    public static RewardType match(Lotto lotto, Lotto winLotto) {
+        int matchCount = lotto.countMatchNumber(winLotto);
+        return findRewardType(matchCount);
+    }
+
+    private static RewardType findRewardType(int matchCount) {
+        return Arrays.stream(RewardType.values()).filter(r -> r.getCountOfMath() == matchCount).findAny().orElse(MISS);
```

Comment:
```suggestion
        return Arrays.stream(RewardType.values())
            .filter(r -> r.getCountOfMath() == matchCount)
            .findAny()
            .orElse(MISS);
```
객체지향 생활체조 원칙에 따르면 한 줄에는 점을 하나만 찍는것을 원칙으로 하고있습니다.😀

[객체지향 생활 체조 원칙](https://blogshine.tistory.com/241)

Reply:
앗 수정하겠습니다!